application/models/LSActiveRecord.php

```diff
@@ -399,7 +399,7 @@ public static function decryptSingleOld($value = ''): string
             $sodium = Yii::app()->sodiumOld;
         }
         // if $value is provided, it would decrypt
-        if ($value) {
+        if (isset($value) && $value !== '') {
```

Comment:
Is this fix on the right function?
By the function name, seems like this function is not supposed to be used.

Reply:
And ?

Reply:
> decryptSingle() and decrypt() are still using empty. That was my comment.

Yes : because null OR à or "" are not an encrypted value :) . 

See : https://github.com/LimeSurvey/LimeSurvey/pull/2738#issuecomment-1323242180